crates/examples/infra/mod.rs

```diff
@@ -416,22 +396,22 @@ pub trait RunDA<
         let memberships = Memberships {
             quorum_membership: <TYPES as NodeType>::Membership::create_election(
                 known_nodes_with_stake.clone(),
-                quorum_election_config.clone(),
+                known_nodes_with_stake.clone(),
                 config.config.fixed_leader_for_gpuvid,
             ),
             da_membership: <TYPES as NodeType>::Membership::create_election(
                 known_nodes_with_stake.clone(),
-                committee_election_config,
+                config.config.known_da_nodes.clone().into_iter().collect(),
                 config.config.fixed_leader_for_gpuvid,
             ),
             vid_membership: <TYPES as NodeType>::Membership::create_election(
                 known_nodes_with_stake.clone(),
-                quorum_election_config.clone(),
+                known_nodes_with_stake.clone(),
                 config.config.fixed_leader_for_gpuvid,
             ),
             view_sync_membership: <TYPES as NodeType>::Membership::create_election(
                 known_nodes_with_stake.clone(),
-                quorum_election_config,
+                known_nodes_with_stake.clone(),
```

Comment:
does this last one need to clone? Not sure what the type is, but we can probably `move` the final invocation.

Reply:
No, good point